docker/all-in-one/entrypoint.sh

```diff
@@ -358,6 +367,12 @@ run_prelaunch_hooks
 
 if [ -n "${DELEGATED_INIT_LOCATION:-}" ]; then
   fetch_and_execute_delegated_payload
+  DELEGATED_INIT_RETURN_CODE=$?
+  if [ "$DELEGATED_INIT_RETURN_CODE" -ne 0 ]; then
```

Comment:
How about exiting if return code is 0 and falling back to the code below, moved out of the `else`? To be DRY, and the execution time may still be relevant?

Reply:
Oh yes 👀 that'd be way better. Giving it a spin